packages/runtime/container-runtime/src/test/createChildDataStoreSync.spec.ts

```diff
@@ -93,7 +93,7 @@ describe("createChildDataStore", () => {
 				},
 			} satisfies Partial<IContainerRuntimeBase> as unknown as IContainerRuntimeBase,
 			deltaManager: new MockDeltaManager(),
-		} satisfies Partial<IFluidParentContext> as unknown as IFluidParentContext;
+		} satisfies Partial<IFluidParentContextPrivate> as unknown as IFluidParentContextPrivate;
```

Comment:
I think this works?
```suggestion
		} satisfies Partial<IFluidParentContextPrivate> as IFluidParentContextPrivate;
```

Reply:
Never mind, because `satisfies` doesn't push back any type info onto the object literal.  Too bad.